src/test/java/org/tikv/raw/RawKVClientTest.java

```diff
@@ -152,7 +164,18 @@ private void prepare() {
     int sz = remainingKeys.size();
     System.out.println("deleting " + sz);
     int base = sz / WORKER_CNT;
-    remainingKeys.forEach(kvPair -> checkDelete(kvPair.getKey()));
+    remainingKeys.forEach(
+        kvPair -> {
+          try {
+            checkDelete(kvPair.getKey());
+          } catch (final TiKVException e) {
+            logger.warn(
+                "kvPair check delete fails: "
+                    + KeyUtils.formatBytes(kvPair.getKey())
+                    + ", "
+                    + KeyUtils.formatBytes(kvPair.getValue()));
```

Comment:
Can we have a function that wraps this `KeyUtils.formatBytes(key) + ", " + KeyUtils.formatBytes(val)` logic?